nucypher/blockchain/eth/actors.py

```diff
@@ -1112,7 +1112,7 @@ def worker_address(self) -> str:
             self.__worker_address = worker_address
 
         if self.__worker_address == NULL_ADDRESS:
-            return NO_WORKER_BONDED.bool_value(False)
+            return NO_WORKER_BONDED(NULL_ADDRESS)
```

Comment:
What's the benefit of having a CS constant with an explicit string value, over using that same string as a constant (even with a different name)? Taking into account that CS constants are not IDE-friendly, which is a UX friction point for readers of the codebase and potential external collaborators.

Reply:
I must point you to the above dialog.  I am not targeting this part of the code - just hacking around the failures here.

Reply:
:heavy_check_mark: - removed